lib/ae_mdw_web/controllers/tx_controller.ex

```diff
@@ -58,7 +61,32 @@ defmodule AeMdwWeb.TxController do
     end
   end
 
-  def read_tx_hash(tx_hash) do
+  #
+  # Private functions
+  #
+  defp add_spendtx_details(data_txs, %{"account" => account}) do
+    Enum.map(data_txs, fn block ->
+      spend_tx_recipient =
+        if block["tx"]["type"] == @type_spend_tx, do: block["tx"]["recipient_id"]
```

Comment:
Sometimes we're using the `(a && b) || c` notation (on the previous file) and some others we're using `if a, do: b, else: c`, we should probably stick to one

Reply:
Yes, for legacy I am keeping the `(a && b) || c` style but for new code and specially here without an `else` was the `if cond, do:`

Reply:
The `(a && b) || c` code above is new code, that line was added by you

Reply:
I mean when creating a new function. Without `else` I'd rather stick with`if cond, do:` rather than `(a && b) || nil`.
Nevertheless, some people dislike this implicit `nil` return in Elixir. I am okay with both approaches. We discuss with everyone in the meeting.

Reply:
used (a && b) || nil until next refactoring